design.md

```diff
@@ -2,12 +2,26 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+We are targetting CS5 students with this design. We assume the user has a basic understanding of programming.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+We chose this design so that users can write more readable picobot code.
+This way, it is easier to interpret and debug!
+
 ## What behaviors are easier to express in your design than in Picobot's original design?  If there are no such behaviors, why not?
 
+It is easier to issue complex commands. In this language, the else statement allows users to write about choices explicitly. "Can I go this way, if I can't, can I go some other way?"
```

Comment:
it might be more complex than the rule definition seen in the original picobot syntax but I'm not convinced that it will be easier for intro students to understand